### PR TITLE
Created ReadMe and added homepage boilerplate

### LINKED ISSUE #4: Setup Initial ReadMe for new contributors.

### PR DESCRIPTION
**Pull request by sarthakkundra**
Added a basic readme for other contributors to know what the project is about. It will also allow them to set it up locally to work on it.

Fixes #4

### CODE REVIEW

File: README.md
Comment (HarshCasper) [inline README.md:1]: Give a more user-friendly title here. `Contributor Website` is just fine. I was having a different name for the same: `Community Website` since we already have a contributor website in existence and making a v2 on a separate repository would be counter-intuitive.

File: README.md
Comment (HarshCasper) [inline README.md:2]: `one-stop-shop` doesn't seem to very appropriate, in my opinion. Instead of guides, you can mention modules.

File: README.md
Comment (HarshCasper) [inline README.md:1]: Give a line break after headings

File: README.md
Comment (HarshCasper) [inline README.md:4]: Give a line break after headings

File: README.md
Comment (HarshCasper) [inline README.md:38]: The link here is dead, I guess

File: README.md
Comment (HarshCasper) [inline README.md:15]: Indent the same

File: README.md
Comment (HarshCasper) [inline README.md:20]: Indent the same

File: README.md
Comment (HarshCasper) [inline README.md:25]: Indent the same

File: README.md
Comment (HarshCasper) [inline README.md:30]: Indent the same

**Review by HarshCasper — CHANGES_REQUESTED** [review]

File: README.md
Comment (sarthakkundra) [reply README.md:1]: I think `Contributor Website` will be better for two reasons :- 
1. That is what the domain name will be.
2. We'll archive the old contributor website repo once this is published.

I'll change the heading. Thanks!

File: README.md
Comment (sarthakkundra) [reply README.md:2]: Sure.

File: README.md
Comment (sarthakkundra) [reply README.md:38]: Will check. If that's the case, I think we need to change it in the About moja global repo too.

**Comment by sarthakkundra** [comment]
@HarshCasper Sorry for such a huge PR 🤣

**Comment by aornugent** [comment]
Hey @sarthakkundra - this is looking great! Tagging @ksocksTR33Z in case there is any imagery/design we can use.

**Comment by aornugent** [comment]
Hey @sarthakkundra - happy for you to merge this when you're ready. Once it's up please also file issues for outstanding work such.

File: website/docusaurus.config.js
Comment (aornugent) [inline website/docusaurus.config.js:56]: Please change this to the moja Slack

**Comment by sarthakkundra** [comment]
@HarshCasper I have done the requested changes mentioned, merging it now without approval because we need to move forward.

**Pull request merged**
